### DISCUSSION TITLE
How to repeat table footer on each page?

### DISCUSSION THREAD

**Discussion by ndyaanzr** [eclipse-birt/birt#1128] [Q&A]
Hi everyone,

It seems that there is no option in birt to repeat table footer on each page like the header. Is there any other way I can do to display the footer?

**Comment by hvbtup** [comment]
I fear not, with one exception.

If you know in advance that all of your rows have equal height and that your table starts at a fixed position (e.g. by explicitly setting the height property for the detail rows and you use set page-break-before = always for the table), then you know in advance how many detail rows will fit onto one page.

Then you can define a table group which will contain e.g. 20 detail rows per group, and use that group's footer instead of or in addition to the table footer.
